### PR TITLE
LPS-184646

### PR DESCRIPTION
**Pull request by ces-totrinh**
Resend PR due to missing code in 8e8052137624b6316da9a97624941a4380a7cd0f
Previous PR: https://github.com/tototrinh/liferay-portal/pull/63
@ces-quanhuynh @locpham97 @ces-luannguyen

### CODE REVIEW

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/configuration/permission/propagation/DLFolderPortletConfigurationPermissionPropagation.java [hunk 1]
```diff
@@ -56,67 +44,28 @@ public String getMessage() {
...
-			_isBulkSelection(portletRequest)) {
-
-			return false;
-		}
+	protected boolean getDefaultPermissionPropagation(
+		long groupId, long classPK) {
 
-		return true;
-	}
+		if ((groupId == classPK) ||
```
Comment (ces-quanhuynh) [inline modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/configuration/permission/propagation/DLFolderPortletConfigurationPermissionPropagation.java:55]: I think swap position of this to (classPK == groupId) is more readable with the second condition.

Comment (tototrinh) [reply modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/configuration/permission/propagation/DLFolderPortletConfigurationPermissionPropagation.java]: Updated code. Thanks.

**Review by ces-quanhuynh — APPROVED** [review]

**Review by ces-luannguyen — APPROVED** [review]

**Comment by ces-totrinh** [comment]
Hi @hhtank 
This PR passed our review. Please help take a look at this. Many thanks.

**Comment by hhtank** [comment]
@tototrinh LGTM, thanks!

**Comment by ces-totrinh** [comment]
Merged. Thanks all.

**Pull request merged**